cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -88,8 +88,13 @@ type Etcd struct {
 type NodeConfiguration struct {
```

Comment:
@errordeveloper That's handled in https://github.com/kubernetes/kubernetes/pull/41417 which builds upon this one